src/Octoliner.cpp

```diff
@@ -2,67 +2,60 @@
 #include <Wire.h>
 
 void Octoliner::begin(uint8_t value) {
-	Wire.begin();
-	pwmFreq(30000);
+    Wire.begin();
+    pwmFreq(30000);
     analogWrite(0, value);
 }
 
-void Octoliner::writeCmdPin(IOcommand command, uint8_t pin, bool sendStop)
-{
-    Wire.beginTransmission( _i2caddress );
+void Octoliner::writeCmdPin(IOcommand command, uint8_t pin, bool sendStop) {
+    Wire.beginTransmission(_i2caddress);
     Wire.write((uint8_t)command);
     Wire.write(pin);
     Wire.endTransmission(sendStop);
 }
 
-void Octoliner::writeCmdPin16Val(IOcommand command, uint8_t pin, uint16_t value, bool sendStop)
-{
-    Wire.beginTransmission( _i2caddress );
+void Octoliner::writeCmdPin16Val(IOcommand command, uint8_t pin, uint16_t value, bool sendStop) {
+    Wire.beginTransmission(_i2caddress);
     Wire.write((uint8_t)command);
     Wire.write(pin);
     uint8_t temp;
     temp = (value >> 8) & 0xff;
-    Wire.write(temp); // Data/setting to be sent to device
+    Wire.write(temp); // sent to device most significant byte of value
```

Comment:
```suggestion
    Wire.write(temp); // Send the most significant byte of `value` to the device
```

Or even simpler: “Send MSB of `value` to the device”. MSB and LSB are widely accepted abbrevs